e2e/e2e_test.go

```diff
@@ -19,12 +19,16 @@ import (
 	_ "github.com/hashicorp/nomad/e2e/hostvolumes"
 	_ "github.com/hashicorp/nomad/e2e/lifecycle"
 	_ "github.com/hashicorp/nomad/e2e/metrics"
+	// _ "github.com/hashicorp/nomad/e2e/migrations"
 	_ "github.com/hashicorp/nomad/e2e/nomad09upgrade"
 	_ "github.com/hashicorp/nomad/e2e/nomadexec"
 	_ "github.com/hashicorp/nomad/e2e/podman"
+	_ "github.com/hashicorp/nomad/e2e/rescheduling"
 	_ "github.com/hashicorp/nomad/e2e/spread"
 	_ "github.com/hashicorp/nomad/e2e/systemsched"
 	_ "github.com/hashicorp/nomad/e2e/taskevents"
+	// _ "github.com/hashicorp/nomad/e2e/upgrades"
+	// _ "github.com/hashicorp/nomad/e2e/vault"
```

Comment:
can just remove these?

Reply:
Yeah, that's for the best... I had them in there as notes to myself but they shouldn't have made it into the PR. Will fix.

Reply:
Fixed in 13f6674